### DISCUSSION TITLE
Code to Install Manim doesn't work

### DISCUSSION THREAD

**Discussion by apple-pii** [ManimCommunity/manim#4076] [Q&A]
Hi, so basically I was trying to install Mainm using the code the website gave me but when I put it into python and tried to run it this happened.......can someone help me? thanks.
<img width="209" alt="Screenshot 2024-12-28 at 11 04 32" src="https://github.com/user-attachments/assets/17e32b44-41c0-4e2b-9dbd-f4a6f2338e69" />

**Comment by uwezi** [comment]
- which operating system?
- which instructions did you follow exactly?
